code/modules/mob/living/silicon/robot/laws.dm

```diff
@@ -26,7 +26,7 @@
 			to_chat(src, SPAN_WARNING("<b>No AI selected to sync laws with, disabling lawsync protocol.</b>"))
 			law_update = FALSE
 
-	to_chat(who, SPAN_WARNING("<b>Obey these laws:</b>"))
+	to_chat(who, SPAN_WARNING("<b>Obey these laws: (No law overrides any other law unless explicitly stated; laws refer to the Stationbound unit and not the player)</b>"))
```

Comment:
I dont know how I feel about a random OOC sentence getting slammed in there, causing a newline and also looking terrible. I would put this at the bottom after laws.

Reply:
I'll try it in a little while, what's important is that it's there and it's visible, and people read and understand it before applying their laws, other than that how it looks/where to put it is secondary so long as it's visible and easy to notice/etc.

Reply:
You were right, it looks a bajillion times better and shouldn't be any less clear FMPOV.
This should be resolved since we're definitely going with this if there aren't any rejections then. 👀